packages/server/src/arpa_reporter/lib/outputTemplates.json

```diff
@@ -233,7 +311,7 @@
     ]
   },
   "project18_229233BulkUpload.xlsx": {
-    "version": "2022.12.13",
+    "version": "2022.5.17",
```

Comment:
[sic] Note that this typo in the date is in the treasury template, not introduced by our code.